src/controllers/projectController.ts

```diff
@@ -25,10 +26,36 @@ export class ProjectController implements Disposable {
     }
 
     public async createJavaProject() {
-        const javaVersion: number = await this.getJavaVersion();
-        if (!javaVersion) {
-            return;
+        const projectKinds: QuickPickItem[] = [{
+            label: BuildTool.None,
+            detail: "A project without any build tools",
+        }];
+        if (contextManager.getContextValue(Context.MAVEN_ENABLED)) {
+            projectKinds.push({
+                label: BuildTool.Maven,
+                detail: "Use Maven as the build tool of your Java project",
+            });
+        }
+        const choice: QuickPickItem | undefined = projectKinds.length === 1 ? projectKinds[0] :
+            await window.showQuickPick(projectKinds, {
+                ignoreFocusOut: true,
+                placeHolder: "Select the project build tool",
+            },
+        );
+
+        switch (choice.label) {
```

Comment:
`choice` is possible to be `undefined`, if you have more than one project kinds.